addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -308,11 +466,12 @@ def drop_off_picked(self, continue_batch, location_barcode):
             for pick in pickings:
                 pick_todo = pick
                 pick_mls = completed_move_lines.filtered(lambda x: x.picking_id == pick)
+
                 if pick._requires_backorder(pick_mls):
                     pick_todo = pick._backorder_movelines(pick_mls)
```

Comment:
Perhaps `_pick_todo` to indicate that we don't need the return value but are only interested in the side effect (backorder creation). BTW, why `_backorder_movelines` with the underscore if we're calling it from `StockPickingBatch`?